.release-it.json

```diff
@@ -0,0 +1,23 @@
+{
+    "hooks": {
```

Comment:
Why is this `hooks` property here? This looks like something that can be captured by an NPM action (in this application's case, by [main.yml](https://github.com/Amsterdam/ee-docs-importer/blob/develop/.github/workflows/main.yml#L29)